src/esp/bindings_js/index.js

```diff
@@ -26,13 +31,63 @@ function preload(url) {
   return file;
 }
 
+function preloadPhysConfig(url) {
+  let emDataHome = "/data";
+  FS.mkdir(emDataHome);
+
+  let file = url;
+  if (url.indexOf("http") === 0) {
+    const splits = url.split("/");
+    file = splits[splits.length - 1];
+  }
+  FS.createPreloadedFile(emDataHome, file, dataHome.concat(url), true, false);
+
+  let emObjHome = emDataHome.concat("/objects");
+  FS.mkdir(emObjHome);
+
+  // TODO Need to loop through the objects directory on the server (`phys/objects/*`) and put all of the glbs onto the client
+  var objects = fileBasedObjects["objects"];
+  for (let objectIdx in objects) {
+    let objectName = objects[objectIdx]["objectName"];
+    let objectHandle = objects[objectIdx]["objectHandle"];
+    let physicsProperties = objects[objectIdx]["physicsProperties"];
+    let renderMesh = objects[objectIdx]["renderMesh"];
+
+    FS.createPreloadedFile(
+      emObjHome,
+      objectName,
+      dataHome.concat(renderMesh),
+      true,
+      false
+    );
+    FS.createPreloadedFile(
+      emObjHome,
+      objectHandle,
+      dataHome.concat(physicsProperties),
+      true,
+      false
+    );
+  }
+
+  return emDataHome.concat("/".concat(file));
+}
+
 Module.preRun.push(() => {
   let config = {};
   config.scene = defaultScene;
   buildConfigFromURLParameters(config);
   window.config = config;
   const scene = config.scene;
   Module.scene = preload(scene);
+
+  var physicsConfigFile = window.config.defaultPhysConfig;
```

Comment:
Do we reference the window explicitly anywhere else? Best to avoid referencing it since it can limit where the JS can be deployed (like to node).

Reply:
AH nvm, that's our bad I just realized as we do set the window in the line before, disregard that comment. That should be added as a todo on our side.